src/popup/component/LedgerConnected/index.js

```diff
@@ -30,11 +31,29 @@ export class LedgerConnected extends React.Component {
       </div>
     )
   }
+
+  renderSpicalStyle = (list,outerIndex) => {
+    return (
+      <p className={"wallet-tip-description"} key={outerIndex+""}>
+        {
+          list.map((item, index) => {
+            if (item.type === "common") {
+              return (<span key={index + ""}>{item.showStr}</span>)
+            } else {
+              return (<span key={index + ""} className={"tips-spical"}>{item.showStr}</span>)
```

Comment:
I think this is too complex to maintain. Just use `Trans` element from react-i18next

Reply:
The prompts displayed here are passed over. If you use Trans, you may need to pass components over. I don’t find it difficult to maintain, because when you want a special display, you can directly add special symbols. Trans is the same, and The bolding here is not a variable, I don’t think there is any difference between using Trans and the current one